python/metatensor-core/setup.py

```diff
@@ -102,7 +102,7 @@ def run(self):
             check=True,
         )
         subprocess.run(
-            ["cmake", "--build", build_dir, "--target", "install"],
+            ["cmake", "--build", build_dir, "--parallel", "--target", "install"],
```

Comment:
this does not do anything, since we are not compiling any C++ code. Does not hurt to have it though!

Reply:
Ah I assume it just calls the Rust compiler?